runtime/Makefile

```
@@ -1,6 +1,6 @@
 PYTHON := python3
-C_COMPILER?=clang
-CXX_COMPILER?=clang++
+C_COMPILER?=gcc-10
+CXX_COMPILER?=g++-10
```

What's blocking from using clang?
```suggestion
C_COMPILER?=clang
CXX_COMPILER?=clang++
```